app/src/main/java/cm/aptoide/pt/app/view/AppViewFragment.java

```diff
@@ -264,6 +270,25 @@
   private View flagThisAppSection;
   private View collapsingAppcBackground;
 
+  //wallet promotion
+  private View promotionView;
+  private View walletPromotionDownloadLayout;
+  private View walletPromotionClaimLayout;
+  private Button walletPromotionClaimButton;
+  private View walletPromotionInstallDisableLayout;
+  private Button walletPromotionInstallDisableButton;
+  private TextView walletPromotionTitle;
+  private TextView walletPromotionMessage;
+  private View walletPromotionButtonsLayout;
+  private Button walletPromotionCancelButton;
+  private Button walletPromotionDownloadButton;
+  private ProgressBar downloadWalletProgressBar;
+  private TextView downloadWalletProgressValue;
+  private ImageView cancelWalletDownload;
+  private ImageView pauseWalletDownload;
+  private ImageView resumeWalletDownload;
+  private PublishSubject<PromotionEvent> promotionAppClick;
+
```

Comment:
Note: similarAppcAppClick and similarAppcAppsVisibilitySubject are not being used. It is not related to this PR but if you could remove it.. Plus they are not being released on the onDestroy